.stylelintrc.json

```diff
@@ -14,6 +14,9 @@
     }],
     "selector-pseudo-class-no-unknown": [true, {
       "ignorePseudoClasses": ["global"]
-    }]
+    }],
+    "function-url-quotes": null,
+    "scss/operator-no-newline-after": null,
+    "selector-not-notation": null
```

Comment:
Is this a temp thing or do we expect to turn these off now in our stylelint config?

Reply:
It is temporary, but couldn't add the comment in the json file 😞